src/main/java/org/sagebionetworks/web/client/widget/entity/editor/TableQueryResultWikiEditor.java

```diff
@@ -33,6 +33,13 @@ public void configure(WikiPageKey wikiKey, Map<String, String> widgetDescriptor,
 			isQueryVisible = Boolean.parseBoolean(isQueryVisibleString);
 		}
 		view.setQueryVisible(isQueryVisible);
+		
+		boolean isTableOnly = false;
```

Comment:
Added a new option to "show a basic table only" in the wiki editor for a Synapse Table/View